src/patternfly/components/Tooltip/tooltip.scss

```diff
@@ -1,24 +1,24 @@
 // @debug $tooltip; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
-
-.#{$tooltip} {
+:root {
   // Component variables
   --#{$tooltip}--MaxWidth: #{pf-size-prem(300px)};
-  --#{$tooltip}--BoxShadow: var(--#{$pf-global}--BoxShadow--md);
-
+  --#{$tooltip}--BoxShadow: var(--pf-t--global--box-shadow--md);
+  
   // Content variables
-  --#{$tooltip}__content--PaddingTop: var(--#{$pf-global}--spacer--sm);
-  --#{$tooltip}__content--PaddingRight: var(--#{$pf-global}--spacer--sm);
-  --#{$tooltip}__content--PaddingBottom: var(--#{$pf-global}--spacer--sm);
-  --#{$tooltip}__content--PaddingLeft: var(--#{$pf-global}--spacer--sm);
-  --#{$tooltip}__content--Color: var(--#{$pf-global}--Color--light-100);
-  --#{$tooltip}__content--BackgroundColor: var(--#{$pf-global}--BackgroundColor--dark-100);
-  --#{$tooltip}__content--FontSize: var(--#{$pf-global}--FontSize--sm);
-
+  --#{$tooltip}__content--PaddingTop: var(--pf-t--global--spacer--sm);
+  --#{$tooltip}__content--PaddingRight: var(--pf-t--global--spacer--md);
+  --#{$tooltip}__content--PaddingBottom: var(--pf-t--global--spacer--sm);
+  --#{$tooltip}__content--PaddingLeft: var(--pf-t--global--spacer--md);
+  --#{$tooltip}__content--Color: var(--pf-t--global--text--color--inverse);
+  --#{$tooltip}__content--BackgroundColor: var(--pf-t--global--background--color--inverse--default);
+  --#{$tooltip}__content--FontSize: var(--pf-t--global--font--size--body--sm);
+  --#{$tooltip}__content--BorderRadius: var(--pf-t--global--border--radius--small);
+  
   // Arrow variables
   --#{$tooltip}__arrow--Width: #{pf-size-prem(15px)};
   --#{$tooltip}__arrow--Height: #{pf-size-prem(15px)};
-  --#{$tooltip}__arrow--BackgroundColor: var(--#{$pf-global}--BackgroundColor--dark-100);
-  --#{$tooltip}__arrow--BoxShadow: var(--#{$pf-global}--BoxShadow--md);
+  --#{$tooltip}__arrow--BackgroundColor: var(--pf-t--global--background--color--inverse--default);
+  --#{$tooltip}__arrow--BoxShadow: var(--pf-t--global--box-shadow--md);
   --#{$tooltip}__arrow--m-top--TranslateX: -50%;
   --#{$tooltip}__arrow--m-top--TranslateY: 50%;
```

Comment:
The arrows feel too close to the rounded corners to me. Can the y translations be adjusted to move them in a little without causing other problems?

Otherwise, it looks great!